sdk/messaging/azeventhubs/internal/blob/zc_sas_account.go

```diff
@@ -27,7 +27,7 @@ type AccountSASSignatureValues struct {
 	ResourceTypes string      `param:"srt"` // Create by initializing AccountSASResourceTypes and then call String()
 }
 
-// Sign uses an account's shared key credential to sign this signature values to produce
+// Sign uses an account's SharedKeyCredential to sign this signature values to produce
```

Comment:
This is just a copy of the library and will get auto-updated. Can you remove these changes from this PR?

Reply:
Thank you for bringing this to my attention! Fixed now :)